README.md

```diff
@@ -1,146 +1,195 @@
 | --- | --- | --- | --- |
 
-The MSAL library for Android gives your app the ability to use the [Microsoft Cloud](https://cloud.microsoft.com) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft Accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. The library also supports [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/).
+The MSAL library for Android gives your app the ability to use the [Microsoft Cloud](https://cloud.microsoft.com) by supporting [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) and [Microsoft accounts](https://account.microsoft.com) in a converged experience using industry standard OAuth2 and OpenID Connect. The library also supports [Azure AD B2C](https://azure.microsoft.com/services/active-directory-b2c/).
 
 [![Version Badge](https://img.shields.io/maven-central/v/com.microsoft.identity.client/msal.svg)](http://repo1.maven.org/maven2/com/microsoft/identity/client/msal/)
 [![Build Status](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android.svg?branch=master)](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-android)
 
 
-##Introduction
+## Introduction
 
-### What's new with this library?
+### What's new?
 
-** need help filling this in
-  -Features lib supports
-  -updates from PP to ga
-  -ADAL to MSAL changes
+> Note: I suggest that we link to our docs for many of these things
 
-### Recommended Usage
+- MSAL Android is now generally available with MSAL 1.0
+- Supported Authorities
+  - Microsoft identity platform (also known as the Azure Active Directory v2 Endpoint)
+  - Azure Active Directory B2C
+- Microsoft authentication broker support
+  - Supports enterprise scenarios including:
+    - Device Registration
+    - Device Management
+    - Intune App Protection
+    - Device Single Sign On
+- Introduction of Single and Multi Account Public Client Applications
+- IAccount and access to claims
+- Support for synchronous methods from worker threads
+- Improved configuration and control of your PublicClientApplication using configuration file
+- Implemented using AndroidX
 
-* supported devices, API levels (need help here)
+### Migrating from ADAL
 
+You can review the ADAL to MSAL migration guide here
 
-## Installation
+### Migrating from preview versions of MSAL
 
-For a full example, checkout the [quickstart](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-v2-android) and [code sample](https://github.com/Azure-Samples/ms-identity-android-java).
+You can review the MSAL Preview to GA release guide here
```

Comment:
Just so that we don't forget: we'll need to call out MSAL devs to handle MsalClientException with error code = "ErrorStrings.UNSUPPORTED_BROKER_VERSION". (Telling the user to update their broker app).

Reply:
The current error msg is 
                    "The protocol versions between the MSAL client app and broker are not compatible."

Should we turn this into something more user-friendly?